packages/runner-ct/src/lib/state.ts

```diff
@@ -203,6 +237,7 @@ export default class State {
   }
 
   runMultiMode = async () => {
+    const eventManager = require('./event-manager')
```

Comment:
I had to move this here otherwise the unit test completely blows up due to side effects in `event-manager` :shrug: